### PR TITLE
Feat: Portfolio Section

### PR DESCRIPTION
**Pull request by cp-shruti-s**
**Video:**
[portfolioSection.zip](https://github.com/canopas/canopas-website/files/10410256/portfolioSection.zip)

**Desktop Performance:**
Screenshot from 2023-01-18 17-24-05

**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [x] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

### CODE REVIEW

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                : portfolio.id == 5
+                ? 'tw-absolute tw-w-[18.66667%]  md:tw-w-[17.6667%] lg:tw-w-[19.666%] xl:tw-w-[20.666%] tw-h-full tw--top-[75px] sm:tw-top-[52px] md:tw--top-[124px] lg:tw--top-[193px] xl:tw--top-[244px] 2xl:tw--top-[256px] tw-left-[52px] md:tw-left-[126px] lg:tw-left-[169px] xl:tw-left-[200px] 2xl:tw-left-[207px] '
+                : '',
+            ]"
+            class="tw-hidden md:tw-block"
+          >
+            <img
+              v-if="portfolio.video"
+              :src="portfolio.video"
+              alt="togness-video"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:73]: Let's use this from data as its not specific for togness only

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+                  activeIndex == 4 ||
+                  activeIndex == 5 ||
+                  activeIndex == 7
+                ? 'tw-px-[50px] '
+                : '',
+            ]"
+            :src="portfolio.image[3]"
+            :srcset="`${portfolio.image[0]} ${portfolio.imageRenderSize[0]}, ${portfolio.image[1]} ${portfolio.imageRenderSize[1]}, ${portfolio.image[2]} ${portfolio.imageRenderSize[2]}, ${portfolio.image[3]} ${portfolio.imageRenderSize[3]}`"
+            sizes="(min-width: 992px) 45vw, 100vw"
+            class="tw-w-full tw-h-full tw-object-cover"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:57]: The last image is straightened, Fix it

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+          >
+            {{ portfolio.description }}
+          </div>
+        </div>
+
+        <div
+          class="tw-w-full"
+          :class="[
+            portfolio.id == 8
+              ? 'tw-h-full tw-p-0'
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:89]: Let's make height class common as its same at all conditions

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+          />
+          <aspect-ratio
+            :class="[
+              portfolio.id == 2
+                ? 'tw-absolute  tw-w-[18.66667%]  md:tw-w-[23.777%] lg:tw-w-[28.666%] xl:tw-w-[30.666%] tw-h-full tw--top-[75px] sm:tw-top-[52px] md:tw--top-[131.5px] lg:tw--top-[219px] xl:tw--top-[285px]  tw-left-[52px] md:tw-left-[55px] lg:tw-left-[58px] xl:tw-left-[59.4px]'
+                : portfolio.id == 5
+                ? 'tw-absolute tw-w-[18.66667%]  md:tw-w-[17.6667%] lg:tw-w-[19.666%] xl:tw-w-[20.666%] tw-h-full tw--top-[75px] sm:tw-top-[52px] md:tw--top-[124px] lg:tw--top-[193px] xl:tw--top-[244px] 2xl:tw--top-[256px] tw-left-[52px] md:tw-left-[126px] lg:tw-left-[169px] xl:tw-left-[200px] 2xl:tw-left-[207px] '
+                : '',
+            ]"
+            class="tw-hidden md:tw-block"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:68]: Why `hidden` in mobile devices?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/PortfolioSection.vue (hunk 4 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/PortfolioSection.vue:64]: As i have given manually  absolute position to video , so I am unable to fix video in sm devices .

**Review by cp-priyanka-g — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**